src/pages/markets/static/content/_basket.tsx

```diff
@@ -8,39 +8,27 @@ import PractiseIcon from 'images/svg/markets/aim-new.svg'
 import TradeIcon from 'images/svg/markets/trade-new.svg'
 import WithdrawIcon from 'images/svg/markets/deposit-and-withdrawal-new.svg'
 
-export type SimpleStepContentElement = {
-    header: ReactElement
-    text: ReactElement
-    icon: ReactElement
-}
-
 export type BasketIndicesContent = {
```

Comment:
Please create a global type for this because we are using that in all of our contents